src/module/api/timeline/create/index.ts

```diff
@@ -25,9 +25,9 @@ export async function Create(newTimeline: ICreateTimeline) {
   obj.getMetadataMap().set(key.VentureID, newTimeline.ventureId);
   obj.setProperty(objProperty);
 
-  if (!newTimeline.membersWrite) {
-    obj.getMetadataMap().set(key.PermissionModel, "reader");
-  }
+  obj
+    .getMetadataMap()
+    .set(key.PermissionModel, newTimeline.membersWrite ? "writer" : "reader");
```

Comment:
By requiring a value of "writer" before allowing members to create updates, timelines created before this feature was added will default to "reader".